pages/mi/mi-10-steps.md

```diff
@@ -108,7 +89,7 @@ Follow the guide at [Create More Issues and Pull Requests](mi-issues-and-prs.md)
 
 ## Step 9 - Kotlin Tour
 
-myPlanet use Kotlin as the development language. If you have no or little prior exposure to Kotlin, please go over the [Kotlin crash-course](https://developer.android.com/kotlin/learn) on Android Developers to gain some familiarity.
+myPlanet uses Kotlin as the development language. If you have no or little prior exposure to Kotlin, please go over the [Kotlin crash-course](https://developer.android.com/kotlin/learn) on Android Developers to gain some familiarity.
```

Comment:
It looks like you created the branch `update-step2` from `3515-fix-typo`. Please ensure you follow the steps outlined at [Synchronize Your Repository](https://open-learning-exchange.github.io/#!./pages/mi/mi-github-issues.md#4._Synchronize_Your_Repository) to fetch changes from upstream into your `master` branch first. Then, create a new branch for your new issue.

Typically, branches should be created from `master`, unless there's a specific reason—such as working on a feature in `3515-fix-typo` where someone is stuck, and you decide to help by continuing from their branch.

Another issue I noticed from the [Commits](https://github.com/open-learning-exchange/open-learning-exchange.github.io/pull/3526/commits) tab is that all commits were created using the GitHub web interface. While many of the changes in First Steps involve simple text edits, we also want to see that you can work with Git in the terminal. This is important because, as an intern, you’ll likely need to work on code and test locally beyond what the GitHub interface allows.

Given this, please follow the steps outlined in [Synchronize Your Repository](https://open-learning-exchange.github.io/#!./pages/mi/mi-github-issues.md#4._Synchronize_Your_Repository) to rework this pull request using the terminal/git bash.

Reply:
Got it. Thank you.